### PR TITLE
[KO] Translated updating/accessing maps section into Korean

### PR DESCRIPTION
**Pull request by devleoper**
Related: #201 and #481

Korean Translation is not finished yet! Elixir changes day-by-day so does Elixir School. :laughing:

Even this pull request actually translated just a line, feel free to give a feedback.

@Dalgona @riseshia @malkoG

### CODE REVIEW

**Comment by Dalgona** [comment]
#402 #452

File: ko/lessons/basics/collections.md [hunk 1]
```diff
@@ -148,3 +148,14 @@ iex> %{foo: "bar", hello: "world"}
 iex> %{foo: "bar", hello: "world"} == %{:foo => "bar", :hello => "world"}
 true
 ```
+
+맵에는 또다른 특별한 문법이 있는데, 이로써 애텀 키를 통해 맵 내부를 열람하거나 수정할 수 있습니다.
```
Comment (malkoG) [inline ko/lessons/basics/collections.md]: I think `접근` instead of `열람` is more proper.
`열람` is not used for that context.

**Comment by malkoG** [comment]
:100:

File: ko/lessons/basics/collections.md (hunk 1 above)
Comment (devleoper) [inline ko/lessons/basics/collections.md]: "접근"도 아주 적절한 번역은 아닌 것 같았어요. 읽고-쓸 수 있는 구문을 제공한다는 쪽에서 열람이 더 적절할 것 같다고 생각했는데, 다른 의견 있으세요?

Comment (riseshia) [inline ko/lessons/basics/collections.md]: I prefer `열람` than `접근` XD
these have similiar nuance, but both is not exactly good at this point, so it's just my favor. ~_~

**Comment by Dalgona** [comment]
Good to go :rocket:

**Pull request merged**